tests/caasp/rebootmgr.pm

```diff
@@ -110,7 +110,7 @@ sub run {
     record_info 'Maint-window', 'Test maint-window strategy';
     check_strategy_maint_window;
 
-    if (!check_var('SYSTEM_ROLE', 'microos')) {
+    if (!(check_var('SYSTEM_ROLE', 'microos') || check_var('SYSTEM_ROLE', 'kubeadm'))) {
```

Comment:
@sysrich - shouldn't that be `if (check_var('SYSTEM_ROLE', 'microos'))` now?

Reply:
Indeed :) good catch

Reply:
@kravciak No, you only want the tests to run if it is "anything that isn't microos" but not kubeadm

so the code is right..

Reply:
microos and kubeadm both dont have etcd ;)

Reply:
ah, bad catch - it would disable it on caasp :(

Reply:
```suggestion
    if (!is_caasp('kubic')) {
```

Reply:
let me change into this. It's simpler to understand.